apps/directory/src/components/report-components/ContactInformation.vue

```diff
@@ -3,7 +3,16 @@
     <div class="font-weight-bold mr-1">Contact:</div>
     <div v-if="name">{{ name }}</div>
     <div v-if="contactInformation.email">
-      <a :href="'mailto:' + contactInformation.email">
+      <a
+        :href="'mailto:' + contactInformation.email"
+        @click="
+          trackMatomoEvent(
+            'Contact',
+            'Email Link Click',
+            contactInformation.email
```

Comment:
I understand it can have 4 parameters? Shouldn't we make a typescript object for that payload? and the 'value' can this be anything?

Reply:
Might be nice indeed to have an interface for this. Will do